### PR TITLE
update docker file and fix docker-push action.

### PR DESCRIPTION
**Pull request by ahmetoz**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,12 +1,6 @@
 FROM node:14-alpine3.15
 MAINTAINER Professional Services <ps-dev@commercetools.com>
-
 WORKDIR /app
-
 COPY . /app
-
-RUN apk add --no-cache --virtual .gyp python3 make g++
```
Comment (ahmetoz) [inline Dockerfile:8]: assuming those are needed for bunyan and we are not using it anymore.

File: src/runner.js [hunk 2]
```diff
@@ -14,8 +14,6 @@ async function run() {
   const startDate = new Date()
   const packageJson = await getPackageJson()
   const logger = getLogger()
-  logger.info(`${packageJson.name} started`)
```
Comment (ahmetoz) [inline src/runner.js:17]: duplicate log also in index.js that's why it's removed here to avoid duplication of logs.

Comment (butenkor) [inline src/runner.js:17]: I find it useful to see that the job could start successfully, don't you think?

**Review by lojzatran — APPROVED** [review]

File: src/runner.js (hunk 2 above)
Comment (ahmetoz) [reply src/runner.js:17]: https://github.com/commercetools/commercetools-subscriptions/pull/33#discussion_r858493977

**Review by butenkor — APPROVED** [review]
🥇

File: Dockerfile (hunk 1 above)
Comment (praveenkumarct) [reply Dockerfile:8]: Interesting to see github action is working now without python installation. 
I added this after this failure, https://github.com/commercetools/commercetools-subscriptions/runs/5517795316?check_suite_focus=true

Comment (lojzatran) [reply Dockerfile:8]: It was working before like that: https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/Dockerfile

Comment (praveenkumarct) [reply Dockerfile:8]: Okay, I will check what was the problem in CD deployment script, thanks :)

**Pull request merged**